src/mysql/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@modelcontextprotocol/server-mysql",
+  "version": "0.1.0",
```

Comment:
Since we are currently releasing all servers at the same time, they should all have matching versions.